### PR TITLE
fix(DB/Quest) add missing quest to brewfest npc's

### PR DESCRIPTION
**Pull request by locus313**
## CHANGES PROPOSED:
-  Add quest "Catch the Wild Wolpertinger!" to both Goldark Snipehunter and Glodrak Huntsniper

## ISSUES ADDRESSED:
- Closes none

## TESTS PERFORMED:
- apply sql and restart server
- .event start 24
- checked that both Goldark Snipehunter and Glodrak Huntsniper now provide the quest.
- checked that you only get one Stunned Wolpertinger per use of the Wolpertinger Net
- checked that that you can complete the quest

## HOW TO TEST THE CHANGES:

- apply sql and restart server
- .event start 24
- check that both (.go c 88867) Goldark Snipehunter and (.go c 88932) Glodrak Huntsniper now provide the quest.
- check that you only get one Stunned Wolpertinger per use of the Wolpertinger Net
- check that that you can complete the quest

## KNOWN ISSUES AND TODO LIST:

- [x]  Seem when you use the "Wolpertinger Net" you receive two of the "Stunned Wolpertinger" when it should only be one.

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by locus313** [comment]
@genion18 Was wondering if you might be able to help on this one as well. Trying to pinpoint why the "Wolpertinger Net" is giving two of the "Stunned Wolpertinger". Do you have any suggestions?

**Review by Helias — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1601269592461165174.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+DELETE FROM `creature_questender` WHERE (`quest` = 11117) AND (`id` IN (23486));
+INSERT INTO `creature_questender` (`id`, `quest`) VALUES
+(23486, 11117);
+
+/* fix for Wolpertinger Net */
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 23487;
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 23487) AND (`source_type` = 0) AND (`id` IN (0));
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(23487, 0, 0, 1, 8, 0, 100, 0, 41621, 0, 0, 0, 0, 11, 32906, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Add item on spell cast');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1601269592461165174.sql:25]:
```suggestion
(23487, 0, 0, 1, 8, 0, 100, 0, 41621, 0, 0, 0, 0, 11, 32906, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Wild Wolpertinger - On Spellhit \'Wolpertinger Net\' - Add Item \'Stunned Wolpertinger\'');
```

**Comment by pangolp** [comment]
Tested in the alliance.

WoWScrnShot_122920_201229

WoWScrnShot_122920_201238

WoWScrnShot_122920_201247

WoWScrnShot_122920_201256

WoWScrnShot_122920_201305

**Comment by pangolp** [comment]
Tested on the horde.
The quest works well, however, the character, in the area, performs an animation, as if he were throwing something continuously, no reason why. It has nothing to do with the pull request, but it called my attention to that behavior.

WoWScrnShot_122920_204236

WoWScrnShot_122920_204411

WoWScrnShot_122920_204426

WoWScrnShot_122920_204451

**Comment by locus313** [comment]
@pangolp Thanks! Could you open a issue for the animation bug you noticed. Ill test it later to confirm it?

**Pull request merged**